### PR TITLE
VPN $4.99 Cyber security month sale (Fixes #10539)

### LINKED ISSUE #10539: Update VPN EN landing page for 4.99 Cyber security month sale.
## Description

For Cyber Security Month we want to offer $4.99 / month pricing in EN, CA and UK. This would mean anyone who subscribes in these countries will get no commitment $4.99 pricing for as long as they keep the account. 

Current recommendation will be to change the top of the page to advertise a Cyber Security Month sale.

Copy and UX to be provided by Mary and Elise.

Brief: https://docs.google.com/document/d/1pd38js6cqChH_3Dqx-N3sXZdS_VfSjU5KaK4UEqmShU/edit?usp=sharing

This issue will be updated when more info is available.

---

#### Success Criteria
- [ ] Approved copy from Mary.
- [ ] Approved UX from Elise
- [ ] Send staging to Lauren for review and approval before merge.
- [ ] can be reverted at the end of October

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds a limited time offer $4.99 VPN subscription offer to the hero section of the VPN landing page for Cyber Security month.

Promo is displayed to:
- Countries: US, CA, GB
- Languages: `en`

Promo is behind the switch `SWITCH_VPN_CYBER_SECURITY_OFFER`

## Issue / Bugzilla link
#10539

## Testing
With `SWITCH_VPN_CYBER_SECURITY_OFFER=on` in your `.env`:

- [ ] http://localhost:8000/en-US/products/vpn/ should display the promo.
- [ ] http://localhost:8000/en-CA/products/vpn/ should display the promo.
- [ ] http://localhost:8000/en-GB/products/vpn/ should display the promo.
- [ ] http://localhost:8000/de/products/vpn/ should not display the promo.
- [ ] http://localhost:8000/en-US/products/vpn/?geo=de should not display the promo.

With `SWITCH_VPN_CYBER_SECURITY_OFFER=off` in your `.env`:

- [ ] The promo should not be displayed using any of the above URLs.

Passing test run: https://gitlab.com/mozmeao/www-config/-/pipelines/393402888

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for review, but **do not merge** until we have approval for launch.

File: bedrock/products/templates/products/vpn/landing.html [hunk 1]
```diff
@@ -55,16 +58,40 @@
...
     ) %}
       <div class="js-vpn-scroll-to-pricing">
         {% block hero_cta_change %}
-          <a class="mzp-c-button mzp-t-xl" href="#pricing" data-cta-type="button" data-cta-text="Scroll to pricing" data-cta-position="primary">
-          {{ ftl('vpn-shared-subscribe-link') }}
-          </a>
+          {% if cyber_security_offer %}
+            <p><strong class="t-mark">Limited-time offer</strong> for Cyber Security month:</p>
+            <h2 class="c-limited-offer-heading">
+              <span class="c-limited-offer-price">{{ settings.VPN_LIMITED_OFFER_MONTHLY_PRICE_USD }}</span>/month (save $59.88)
```
Comment (maureenlholland) [inline bedrock/products/templates/products/vpn/landing.html:72]: Assuming copy is still a little up in the air? It seems odd to have the save $59.88 without context (when used in the 12-month plan box, you know it's an annual savings amount)

File: bedrock/products/templates/products/vpn/landing.html [hunk 2]
```diff
@@ -17,6 +17,9 @@
 {% set _utm_campaign = 'vpn-product-page' %}
 {% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
 
+{# Show $4.99/month offer for wave 1 English markets (issue 10539) #}
+{% set cyber_security_offer = switch('vpn-cyber-security-offer') and LANG.startswith('en-') and country_code in ['US', 'CA', 'GB'] %}
```
Comment (maureenlholland) [inline bedrock/products/templates/products/vpn/landing.html:21]: 🎉 `country_code` in template! Exciting!

**Review by maureenlholland — APPROVED** [review]
This is working as expected for me and nicely compartmentalized behind the switch. One copy question for savings text.

File: bedrock/products/templates/products/vpn/landing.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/products/templates/products/vpn/landing.html:72]: I agree, I'll see what our copywriter thinks. Thanks for the suggestion

**Comment by alexgibson** [comment]
This branch is stale and needs a rebase. Going to close it for now, as revisiting would likely need a refactor anyways.

**Pull request closed without merging**